crates/holochain/src/conductor/chc/chc_remote.rs

```diff
@@ -0,0 +1,93 @@
+use std::collections::{HashMap, HashSet};
+
+use ::bytes::Bytes;
+use holo_hash::{ActionHash, EntryHash};
+use holochain_serialized_bytes::{decode, encode};
+use holochain_types::chc::{ChainHeadCoordinator, ChcError, ChcResult};
+use holochain_zome_types::prelude::*;
+use reqwest::Url;
+
+/// An HTTP client which can talk to a remote CHC implementation
+pub struct ChcRemote {
+    actions: ChcRemoteClient,
+    entries: ChcRemoteClient,
+}
+
+#[async_trait::async_trait]
+impl ChainHeadCoordinator for ChcRemote {
+    type Item = SignedActionHashed;
+
+    async fn head(&self) -> ChcResult<Option<ActionHash>> {
+        let response = self.actions.get("/head").await?;
```

Comment:
Are these part of the assumed future api, or are these documented somewhere?

Reply:
Part of the assumed API. I guess I should make it clear that this is just a starting point and that nobody should be designing a CHC to match these endpoints.